src/manifest.json

```diff
@@ -2,12 +2,12 @@
   "manifest_version": 2,
   "name": "__MSG_extensionName__",
   "default_locale": "en_US",
-  "version": "17",
+  "version": "19",
   "author": "Firefox",
   "applications": {
     "gecko": {
       "id": "secure-proxy@mozilla.com",
-      "strict_min_version": "68.0.1",
+      "strict_min_version": "71",
```

Comment:
Self reminder to verify min version on the private guardian repo’s updates.json once we have a PR there.

Reply:
Good point!